src/jagadeeshMJun20/ConvertCaseInStringArray.java

```diff
@@ -0,0 +1,34 @@
+/*Assignement 29 : 17th July 2020
+
+WAP to convert case in all elements of array. 
+Input : 
+String[] array = {"AaVruti", "NiKhil", "AdiTi", "MAhesh", "ArchaNA", "jaGAdeesh"};
+
+Output:
+array = {"aAvRUTI", "nIkHIL","aDItI", "maHESH","aRCHAna","JAgaDEESH"};
+ */
+package jagadeeshMJun20;
+public class ConvertCaseInStringArray {
+	void displayOutput(String[] names) {
+		String outputNames[] = { "", "", "", "", "", "" };
```

Comment:
Here if you want to create new array then please write like this String outputNames[]  = new String[names.length];
String outputNames[] = { "", "", "", "", "", "" };
here problem is if I have added one more name in names then you have to assign index manually.